Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/BaseMousePointer.cs

```diff
@@ -26,23 +25,29 @@ public class DefaultMousePointer : BaseControllerPointer, IMixedRealityMousePoin
         [Tooltip("Should the mouse cursor be hidden when no active input is received?")]
         private bool hideCursorWhenInactive = true;
 
-        /// <inheritdoc />
+        /// <summary>
+        ///  Should the mouse cursor be hidden when no active input is received?
+        /// </summary>
         public bool HideCursorWhenInactive => hideCursorWhenInactive;
 
         [SerializeField]
         [Range(0.01f, 1f)]
         [Tooltip("What is the movement threshold to reach before un-hiding mouse cursor?")]
         private float movementThresholdToUnHide = 0.1f;
 
-        /// <inheritdoc />
+        /// <summary>
+        /// What is the movement threshold to reach before un-hiding mouse cursor?
```

Comment:
nit: it's actually a bit weird that we put the comments in question format - this should be a description: "Movement threshold to reach before un-hiding the mouse cursor"

same applies for the other comments and tooltips

Reply:
speaking of this portion of the code. What about removing hideCursorWhenInactive and instead check for hideTimeout > 0?

Reply:
if we do that we loose the option to hide the cursor immediately after movement stops. you'd have to set some value in there to trigger the hide. I wouldn't change the behavior for now, also this would mean another breaking change

Reply:
Did not thing anyone would want that, but true. reverted it

Reply:
yeah you're right it's unlikely someone wants it, but i wouldn't remove it for now :) thanks!